sdk/python/teaclave.py

```diff
@@ -222,10 +224,12 @@ class FunctionOutput:
     Args:
         name: Name of output data.
         description: Description of the output data.
+        option: [Default: False] Data owners do not need to register the data.
     """
 
-    def __init__(self, name: str, description: str):
+    def __init__(self, name: str, description: str, option=False):
```

Comment:
Probably using `optional = False` could be clearer and it is also consistent with the usage in TOML files. thanks.

Reply:
@qinkunbao, I also suggest to use `optional` instead of `option` for all definitions.

Reply:
Sounds good!